packages/web/src/form/form.js

```diff
@@ -66,15 +66,22 @@ const inputTagProps = (props) => {
   // any errors on this field
   const validationError = errors[props.name]
 
-  // get errorStyle/errorClassName and replace style/className if present
-  const { errorClassName, errorStyle, ...tagProps } = props
+  // get error/validStyle and error/validClassName and replace style/className if present
+  const { errorClassName, errorStyle, validClassName, validStyle, ...tagProps } = props
   if (validationError) {
     if (errorClassName) {
       tagProps.className = errorClassName
     }
     if (errorStyle) {
       tagProps.style = errorStyle
     }
+  } else if (!validationError && props.value) {
```

Comment:
I see what you're getting at here. I think `props.value` might be the wrong thing to check for (just can't get the validation classes to show up in my app yet&mdash;let me know if I'm doing something wrong though / when I'm supposed to see them). I'm not sure what the right thing is yet; will dig in this weekend.